packages/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -22,6 +25,11 @@ export interface TinyliciousClientProps {
 	 * Optional. A logger instance to receive diagnostic messages.
 	 */
 	logger?: ITelemetryBaseLogger;
+
+	/**
+	 * Optional. Options for the runtime of containers created with the {@link TinyliciousClient}.
+	 */
+	runtimeOptions?: IContainerRuntimeOptions;
```

Comment:
Coordinate with @scottn12 on this (see his recent changes for overlap).

I'm against exposing these options here.  See #12275 for more discussion.

Reply:
i'm also not sure why this is necessary to fix the issue at hand? seems like it is uncessary to bundle with the clone deep change

Reply:
Not necessary to fix the issue, but to make the corresponding e2e test less hacky (avoid reaching into private props of the Container to extract the loader so it can be reused). Moving the test to be directly on Container and the override of client options might make the current test unnecessary so this would be a moot point.